connector/scrapers/__init__.py

```
@@ -0,0 +1,2 @@
+from .kutis_scraper import KutisScraper
```

Ruff (0.11.9)

`.kutis_scraper.KutisScraper` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)